hoot-services/src/main/java/hoot/services/controllers/grail/GrailResource.java

```diff
@@ -819,13 +795,10 @@ public Response pullRailsPortToDb(@Context HttpServletRequest request,
         advancedUserCheck(user);
 
         String layerName = reqParams.getInput1();
+
         String jobId = UUID.randomUUID().toString().replace("-", "");
         File workDir = new File(TEMP_OUTPUT_PATH, "grail_" + jobId);
 
-        if (DbUtils.mapExists(layerName)) {
-            throw new BadRequestException("Record with name : " + layerName + " already exists.  Please try a different name.");
-        }
-
```

Comment:
Comment this out to allow overwrite as we're now using this endpoint to refresh a grail reference layer after a changeset affecting it is uploaded